api/go.mod

```diff
@@ -8,6 +8,7 @@ require (
 	github.com/go-test/deep v1.0.2-0.20181118220953-042da051cf31
 	github.com/hashicorp/errwrap v1.0.0
 	github.com/hashicorp/go-cleanhttp v0.5.1
+	github.com/hashicorp/go-hclog v0.12.0
```

Comment:
Just curious -- I don't see where this is imported in changes in the code in the api dir below. What's bringing this in?

Reply:
I'm guessing this is a relic of a change I made and then later backed out.  I can remove it.